### PR TITLE
Updates for 4.20

### PR DESCRIPTION
**Pull request by andygup**
Updates esm repos.
Adds deprecation notice to samples showing local builds of arcgis-js-api with dojo and requireJS.

### CODE REVIEW

File: esm-samples/jsapi-angular-cli/README.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ For additional information, see the [Build with ES modules](https://developers.a
 
 Currently, due to limitations in TypeScript, the APIs [autocasting](https://developers.arcgis.com/javascript/latest/programming-patterns/#autocasting) functionality works best in non-TypeScript applications. No changes are required if you are already using the API without any TypeScript build errors.
 
+Minimum required version is `~4.2.3`.
```
Comment (dasa) [inline esm-samples/jsapi-angular-cli/README.md:43]: Does it work with 4.3? If not, then maybe this should be more specific?

**Review by dasa — APPROVED** [review]

File: esm-samples/jsapi-angular-cli/README.md (hunk 1 above)
Comment (andygup) [reply esm-samples/jsapi-angular-cli/README.md:43]: Nope it doesn't. For Angular 12: TypeScript >=4.2.3 and <4.3.0. I'll update the text.

**Review by odoe — APPROVED** [review]
👍

**Comment by odoe** [comment]
Can you remove a couple of lines in `4.x/amd/dojo/build.profile.js` with the PR?
https://github.com/Esri/jsapi-resources/blob/master/4.x/amd/dojo/build.profile.js#L105
https://github.com/Esri/jsapi-resources/blob/master/4.x/amd/dojo/build.profile.js#L119

thanks!

**Comment by andygup** [comment]
@odoe 2 lines have been removed.

**Review by odoe — APPROVED** [review]

**Pull request merged**